docs/chapters/baseplatform/building-PELUX-sources.rst

```diff
@@ -27,8 +27,12 @@ Building locally
 
 Create a directory for the PELUX build. Instruct repo tool to fetch a manifest
 using the command `repo init`. In this context, branch denotes what branch of
-git repo `pelux-manifests` to use. Then make repo tool fetch all sources using
-the command `repo sync`.
+git repo `pelux-manifests` to use. If you want to use a released version of
+PELUX, use the SHA id of the commit pointed to by the release tag (or the branch
```

Comment:
Isn't there a way to point directly to the release tag? Maybe we should use branches then instead because we would like to commit patches and anyways a branch would be in place at same point for the release right?

Reply:
Well, the release process specifies branches for the major releases, meaning that minor releases would go into the same branch. Or actually it doesn't specify that - but it should.

This means that looking at a branch means you would always get the latest, which might not be the release you are looking for. Let's say you want PELUX 1.1 but the latest on the 1.0 branch is 1.4.

Also unfortunately the repo tool can't resolve tags :(

Reply:
Ah I see, well in order to avoid branch clutter it makes sense to have tags and a 1.X branch.